src/test/java/com/sggc/controllers/SggcControllerIT.java

```diff
@@ -0,0 +1,220 @@
+package com.sggc.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sggc.errors.ApiError;
+import com.sggc.exceptions.TooFewSteamIdsException;
+import com.sggc.exceptions.UserHasNoGamesException;
+import com.sggc.exceptions.VanityUrlResolutionException;
+import com.sggc.models.sggc.SggcResponse;
+import com.sggc.models.steam.request.GetCommonGamesRequest;
+import com.sggc.services.GameService;
+import com.sggc.services.UserService;
+import com.sggc.services.VanityUrlService;
+import com.sggc.validation.ValidationResult;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static com.sggc.services.VanityUrlService.VANITY_URL_NOT_ALPHANUMERIC_ERROR_MESSAGE;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(controllers = SggcController.class)
+public class SggcControllerIT {
+
+    public static final String SGGC_CONTROLLER_URI = "/api/sggc/";
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private ObjectMapper mapper;
+    @MockBean
+    private UserService userService;
+    @MockBean
+    private VanityUrlService vanityUrlService;
+    @MockBean
+    private GameService gameService;
+
+    @Test
+    @DisplayName("Given a request with less than two Steam id's when validating the request body then a 400 error will be returned with an appropriate message")
```

Comment:
"Given a request with less than two Steam ID's, when validating the request body, then a 400 error will be returned with an appropriate message."